bin/pt-online-schema-change

```diff
@@ -12168,6 +12191,15 @@ L<"--print"> and verify that the triggers are correct.
 
 =back
 
+=item --[no]check-foreign-keys
+
+default: yes
+
+Check for self referencing foreing keys. Currently self referencing FKs are
+not fully supported so to prevent errors, this program wont't run if the table
```

Comment:
Change line to
"not full supported, so, to prevent errors, this program won't run if the table"